vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/DialogWindow.java

```diff
@@ -0,0 +1,39 @@
+package life.qbic.datamanager.views.general;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.dialog.Dialog;
+/**
+ * <b>Dialog to create something</b>
```

Comment:
NIT: If this is a dialog solely for creation we can be more explicit with the naming. However I'd argue that this is not the intention 
```suggestion
 * <b>Dialog window providing the possibility to confirm or cancel via user triggered events</b>
```